BUILD.gn

```diff
@@ -175,23 +176,40 @@ process_version("generate_version") {
 copy("brave_dist_resources") {
   sources = [ "$root_out_dir/version" ]
 
-  if (!is_mac) {
-    sources += [
-      "$root_out_dir/brave_resources.pak",
-      "$root_out_dir/brave_100_percent.pak",
-      "$root_out_dir/brave_200_percent.pak",
-      "$root_out_dir/$brave_exe"
-    ]
-  }
-
   deps = [
     ":brave",
     ":generate_version",
     ":brave_license_dist_resources",
     ":packed_resources",
     "build/$target_os:brave",
+    "//third_party/icu:icudata",
   ]
 
+  if (!is_mac) {
+    sources += [
```

Comment:
it seems like we have too many pak files here, they should be consolidated into 3 using repack if they aren't already (brave/100/200)

Reply:
Yep right. Created issue for this (https://github.com/brave/brave-browser/issues/672).